composer.json

```diff
@@ -54,8 +54,6 @@
         "drupal/wysiwyg_template": "^2.0@beta",
         "drupal/dynamic_entity_reference": "^1.7",
         "drupal/scheduled_transitions": "^1.0",
-        "drupal/content_moderation_scheduled_updates": "^1.0",
```

Comment:
@vincent-gao This can't be done on a project until [tide_core_update_8017](https://github.com/dpc-sdp/tide_core/pull/128/files#diff-614ec2ea4dd80b41606479ce51b1dea5) is confirmed to have been deployed on a project. I'm not sure if there's a way to to flag these as deprecated dependencies at a particular release number.

I've opened up an issue to track this. See my comments there and if you have any ideas add them to the comments. 🙏